chart/kubeapps/values.yaml

```diff
@@ -8,6 +8,10 @@
 #     - myRegistryKeySecretName
 #   storageClass: myStorageClass
 
+## Enable the useHelm3 feature flag to use Kubeapps with Helm3 support.
```

Comment:
```suggestion
## Enable this feature flag to use Kubeapps with Helm 3 support.
```
Suggesting the first change because it's more DRY and the second one because it's (more) correct, I think. At least the Helm people [use that style themselves](https://helm.sh/blog/helm-3-released/).